internetarchive/cli/ia.py

```diff
@@ -61,9 +61,9 @@
 import os
 import sys
 
-from docopt import docopt, printable_usage  # type: ignore
+from docopt import docopt, printable_usage
 from pkg_resources import DistributionNotFound, iter_entry_points
-from schema import Or, Schema, SchemaError
+from schema import Or, Schema, SchemaError  # type: ignore[import]
```

Comment:
Why?

Reply:
`schema` seems not to be typed.

Reply:
Does mypy complain if we do not have this directive?

Reply:
Yes.

```shellsession
$ git clone https://github.com/jjjake/internetarchive && cd internetarchive
$ mypy internetarchive
internetarchive/cli/ia_download.py:68: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
internetarchive/cli/ia_delete.py:50: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
internetarchive/session.py:44: error: Skipping analyzing "requests.packages.urllib3": module is installed, but missing library stubs or py.typed marker
internetarchive/cli/ia_upload.py:74: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
internetarchive/cli/ia_search.py:49: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
internetarchive/cli/ia_metadata.py:61: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
internetarchive/cli/ia_copy.py:42: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
internetarchive/cli/ia.py:66: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
internetarchive/cli/ia.py:66: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
internetarchive/cli/ia_move.py:38: error: Skipping analyzing "schema": module is installed, but missing library stubs or py.typed marker
Found 9 errors in 9 files (checked 26 source files)
```

Reply:
Cool.  I need to wheen myself off of `--ignore-missing-imports`